paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestMultiStepFieldsReloaded.kt

```diff
@@ -223,32 +223,6 @@ class TestMultiStepFieldsReloaded {
         )
     }
 
-    @Test
-    fun testCashAppPay_Fail() {
```

Comment:
Actually, testing these don’t really make a lot of sense for the custom flow. The failure/cancel cases are covered in `TestHardCodedLpms`.